Doc/whatsnew/3.12.rst

```diff
@@ -755,6 +778,13 @@ typing
   or more members may be slower than in Python 3.11. (Contributed by Alex
   Waygood in :gh:`74690` and :gh:`103193`.)
 
+* All :data:`typing.TypedDict` and :data:`typing.NamedTuple` classes now have the
+  ``__orig_bases__`` attribute. (Contributed by Adrian Garcia Badaracco in
```

Comment:
Given that this attribute is undocumented (and I'd prefer it to stay that way, though that's not a _strong_ opinion), could we say something like "this ensures that `types.get_original_bases(typeddict_class)` will now always return the correct result"?

Reply:
Maybe we could say "now work reliably with :func:`types.get_original_bases`", although that's a bit of a vacuous statement as `types.get_original_bases` didn't exist before 3.12 :)

Reply:
Eh, I guess you're right, it's probably fine as it is